src/constants.js

```diff
@@ -25,4 +25,19 @@ const ENVIRONMENTS = {
   PRODUCTION: 'production',
 };
 
-module.exports = { ENVIRONMENTS };
+const SSO_SUB_URI = {
+  SA_AUTH_TOKEN: 'auth/realms/master/protocol/openid-connect/token',
+  REALM_ADMIN: 'auth/admin/realms',
+  USER: 'users',
+  GROUP: 'groups',
+  IDP: 'federated-identity',
+};
+
+const SSO_REQUEST = {
+  CONTENT_TYPE_FORM: 'application/x-www-form-urlencoded',
+  CONTENT_TYPE_JSON: 'application/json',
+  GRANT_TYPE: 'client_credentials',
+  CLIENT_ID: 'admin-cli',
+};
+
+module.exports = { ENVIRONMENTS, SSO_SUB_URI, SSO_REQUEST };
```

Comment:
mixing common js `module.exports` here with `export` syntax in another file `src/libs/sso-utils.js`